src/asm/shared/transform/inverse.rs

```diff
@@ -54,7 +54,7 @@ pub mod test {
   use rand::{random, thread_rng, Rng};
 
   pub fn pick_eob<T: Coefficient>(
-    coeffs: &mut [T], tx_size: TxSize, tx_type: TxType, sub_w: usize,
+    coeffs: &mut [T], tx_size: TxSize, tx_type: TxType, sub_h: usize,
```

Comment:
Do we also need to change the sub_w in L136 which calls this function?

Reply:
I should change its name, so yes.